tests/core/config/startupFocus.js

```diff
@@ -0,0 +1,80 @@
+/* bender-tags: editor */
+// jscs:disable maximumLineLength
+/* bender-ckeditor-plugins: about,basicstyles,bidi,blockquote,clipboard,colorbutton,colordialog,div,enterkey,entities,find,flash,font,format,forms,horizontalrule,image,iframe,indent,justify,link,list,newpage,pagebreak,pastefromword,pastetext,removeformat,resize,toolbar,save,selectall,showblocks,smiley,sourcearea,specialchar,stylescombo,table,templates,undo,wysiwygarea */
+// jscs:enable maximumLineLength
+
+
+( function() {
+	'use strict';
+
+	function botInitializer( setup ) {
```

Comment:
This function initializes editor, but also asserts focus so basically it runs whole, single test. Maybe it should be named something like `assertStartupFocus`?